### PR TITLE
article opinion

### LINKED ISSUE #16: Token Curated Registries: The Curation Protocol That Separates the Wheat from the Chaff

### PR DESCRIPTION
**Pull request by shyblugs**
TCRs_Separating_the_Wheat_from_the_Chaf
closes 16

# New content for Aragon Monthly

### Closing of a referenced Issue
Closes #16 

### Content category
_Article_ 

#### Content type:
_Opinion Piece_ 

### Title
Token Curated Registries
The Curation Protocol That Separates the Wheat from the Chaff

### Description
This is a 930 word article I wrote about TCRs, what they're good for, and how we plan to implement one using Aragon.

### CODE REVIEW

**Comment by Bradymck** [comment]
I Fixed a small typo and removed the following in first paragraph: 'That could be the social proof that demonstrates you’re the person your profile pic says you are.' I feel this needs to be reworded or removed. @shyblugs please take a look and see what you can do to update. Other than that it just needs another read through. Looks good.

File: docs/articles/opinion/TCRs_Separating_the_Wheat_from_the_Chaf.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ It was this quest which led to us exploring and then adopting a Token Curated Re
...
 
-Nevertheless, an autonomous system needs to autonomy, fairness and transparent for all. Cryptocurrency and the blockchain technology powering it are transformative inventions that have the potential to make the web � and thus the world � a better place for everyone. Any system that adds to the credibility of the crypto ecosystem, and which rewards fair players, is a welcome step towards making the space more inclusive, thereby attracting new entrants. Token Curated Registries are one way of helping to achieve that.
+Nevertheless, an autonomous system needs autonomy, fairness and transparancy for all. Cryptocurrency and the blockchain technology powering it are transformative inventions that have the potential to make the web – and thus the world – a better place for everyone. Any system that adds to the credibility of the crypto ecosystem, and which rewards fair players, is a welcome step towards making the space more inclusive, thereby attracting new entrants. Token Curated Registries are one way of helping to achieve that.
```
Comment (Smokyish) [inline docs/articles/opinion/TCRs_Separating_the_Wheat_from_the_Chaf.md]: transparency

File: docs/articles/opinion/TCRs_Separating_the_Wheat_from_the_Chaf.md [hunk 2]
```diff
@@ -32,13 +32,13 @@ It was this quest which led to us exploring and then adopting a Token Curated Re
 BitPoint0x is a dual layer TCR for BitPoint tellers, this enables the network to autonomously certify its tellers are aligned with its values. [Kleros](kleros.io) sits on top providing an escrow and arbitration layer protecting consumers as a final resort.
```
Comment (Smokyish) [inline docs/articles/opinion/TCRs_Separating_the_Wheat_from_the_Chaf.md]: Please add `https://` to the link.

**Comment by Bradymck** [comment]
Unless you want to add anything @shyblugs I think this is a good addition. I market approved.

**Pull request merged**